### PR TITLE
Khwang/login register

### PR DESCRIPTION
**Pull request by khwang**
Make the login/register pages look much less awful.

honeydew

---
This change is [Reviewable](https://reviewable.io/reviews/khwang/honeydew/7)

### CODE REVIEW

**Comment by Radahika** [comment]
LGTM. 
Can you add a before photo for future reference?

File: app/views/devise/registrations/new.html.haml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.container
```
Comment (Radahika) [inline app/views/devise/registrations/new.html.haml]: Add a comment to the top of the page to explain what this file is for.

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -1,26 +0,0 @@
-<h2>Log in</h2>
```
Comment (Radahika) [inline app/views/devise/sessions/new.html.erb]: Same as above.

File: app/views/devise/sessions/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.container
```
Comment (Radahika) [inline app/views/devise/sessions/new.html.haml]: Same as above.

File: app/views/devise/registrations/new.html.haml (hunk 1 above)
Comment (khwang) [inline app/views/devise/registrations/new.html.haml]: Shouldn't be necessary; it's self-documenting from the file location. it's a view for devise (the authentication gem) handling new user registrations (app/views/devise/registrations/new.html.haml)

File: app/views/devise/sessions/new.html.haml (hunk 3 above)
Comment (khwang) [inline app/views/devise/sessions/new.html.haml]: Same

**Pull request merged**